opendata.swiss/ui/pages/datasets/[datasetId]/index.vue

```diff
@@ -73,9 +73,15 @@ const breadcrumbs = computed(() => {
   return result
 })
 
+const seoTitle = computed(() => {
+  if (dataset.value?.title) {
+    return `${dataset.value.title} | ${t('message.header.navigation.datasets')} | opendata.swiss`
+  }
+  return `opendata.swiss | ${t('message.header.navigation.datasets')}`
+})
 
 useSeoMeta({
-  title: `${resultEnhanced.value?.getTitle} | ${t('message.header.navigation.datasets')} | opendata.swiss`,
```

Comment:
This is not really a great fix. Removing the actual dataset/distribution title from meta is not the goal. Instead, if you do what they show in the [docs](https://nuxt.com/docs/4.x/api/composables/use-seo-meta#usage) you shared the other day.

The trick is to set meta after `suspense()`

```diff
+await suspense()

useSeoMeta({
-  title: `${resultEnhanced.value?.getTitle} | ${t('message.header.navigation.datasets')} | opendata.swiss`,
+  title: () => `${resultEnhanced.value?.getTitle} | ${t('message.header.navigation.datasets')} | opendata.swiss`,
})

-await suspense()
```